IntegrationTests/Tests/IntegrationTests/XCBuildTests.swift

```diff
@@ -0,0 +1,286 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+import TSCBasic
+import TSCUtility
+import TSCTestSupport
+
+final class XCBuildTests: XCTestCase {
+  #if os(macOS)
+    func testExecutableProducts() throws {
+        fixture(name: "XCBuild/ExecutableProducts") { path in
+            let fooPath = path.appending(component: "Foo")
+            let binaryPath = fooPath.appending(components: ".build", "x86_64-apple-macosx", "Products")
+
+            try sh(swiftBuild, "--package-path", fooPath, "--enable-xcbuild")
```

Comment:
Can you make the flag to be —buildsystem [native|xcode]

Reply:
Of course!